knowledge-base/importing-fiddler-classic-settings.md

```diff
@@ -0,0 +1,68 @@
+
+The import wizard is accessible through the **Home** pane in Fiddler Everywhere.
+![import configuration options](../images/import-from-classic/import-options.png)
+
+- Fiddler Classic certificate&mdash;Instructs Fiddler Everwhere to use the installed Fiddler Classic certificate authority file. This option is available only if you have not yet installed and trusted the Fiddler Everywhere CA.
+
+- Decrypt HTTPS traffic&mdash;Exports the Fiddler Classic value from **Tools > Options > HTTPS > Decrypt HTTPS Traffic** and imports as value for **Settings > HTTPS > Capture HTTPS Traffic** option in Fiddler Everywhere.
+
+- Ignore server certificate errors&mdash;Exports the Fiddler Classic value from **Tools > Options > HTTPS > Ignore server certificate errors (unsafe)** and imports as value for **Settings > HTTPS > Ignore server certificate errors (unsafe)** option in Fiddler Everywhere.
+
+- Listening proxy port&mdash;Exports the Fiddler Classic value from **Tools > Options > Connection > Fiddler Classic listens on port** and imports as value for **Settings > Connections > Fiddler listens on port** option in Fiddler Everywhere.
+
+- Bypassed URLs&mdash;Exports the Fiddler Classic value from **Tools > Options > Connections > Bypass Fiddler Classic for URLs that starts with** and imports as value for **Settings > Connections > Bypass Fiddler for URLs that starts with** option in Fiddler Everywhere.
+
+- Gateway settings&mdash;Exports the Fiddler Classic value from **Tools > Options > Gateway** and imports as value for **Settings > Gateway** option in Fiddler Everywhere.
+
+- Allow remote computers to connect&mdash;Exports the Fiddler Classic value from **Tools > Options > Connections > Allow Remote Computers to connect** and imports as value for **Settings > Connections > Allow remote devices to connnet** option in Fiddler Everywhere.
+
+## Importing AutoResponder Rules
+
+In Fiddler Everywhere, the Classic's **AutoResponder** (as depicted in the screenshot below) is referred to as **Rules**. You can import only the loaded AutoResponder rules from Fiddler Classic to Fiddler Everywhere. Once imported, you'll be able to modify or delete these rules as needed.
+
+![AutoReposnder rules in Fiddler Classic](../images/import-from-classic/fc-autoresponder.png)
+
+## Importing Filters
+
+Fiddler Classic **Filters** function (refer to the screenshot) at a deeper level, excluding any traffic not specifically requested by the user. In Fiddler Everywhere, such actions are initiated through the **Rules** tab, this is why any loaded Fiddler Classic filters will be imported as separate rules.
+
+![filters in Fiddler Classic](../images/import-from-classic/fc-filters.png)
```

Comment:
I think this is a good place to show a screenshot with where the user can find the imported Fiddler Classic filters in Fiddler Everywhere, and how they will look?

Also, we can add the same note about importing any saved .ffx files in Fiddler Everywhere at any point of time through the "import" button in the Rules tab. Because the wizard will import only the currently loaded Filters.

Reply:
I will consider adding a screenshot at a later point. Currently, I aim to minimize the usage of screenshots related to the Fiddler's UI (due to the changes that require maintenance).